backend/cmd/api/search.go

```diff
@@ -47,6 +47,23 @@ func searchQueryHandler(client *search.Service) func(*gin.Context) {
 	}
 }
 
+func searchKeyHandler(client *search.Service) func(*gin.Context) {
+	return func(c *gin.Context) {
+		var query common.SearchQuery
+		// define default options
+		query.Page = 0
+		err := c.BindJSON(&query)
+		if err != nil {
+			log.L.Error().Err(err)
```

Comment:
IIRC you have to use some form of `.Msg()` in order for the log to actually be logged.